docs/contribute/design_axioms.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Design Axioms
+permalink: /contributing/design-axioms
+---
+
+## 💚 100% Kotlin
+
+Kotlin is the language of choice for components we develop and is the recommended language for application projects integrating components. We make use of Kotlin idioms and official Kotlin extension libraries (e.g. Coroutines).
```

Comment:
nit: Kotlin is the language of choice for the components we develop, and it is the recommended [...]